src/application/modules/cb_query.py

```diff
@@ -1,9 +1,12 @@
 from cb_utilities import *
+import urllib
+import re
+import hashlib
 
 def _get_query_metrics(user, passwrd, node_list, cluster_name=""):
     '''Gets the metrics for the query nodes'''
-    query_info = {}
-    query_info['metrics'] = []
+    # first get the system:completed_request entries for the past minute
+    query_info = _get_completed_query_metrics(user, passwrd, node_list, cluster_name)
```

Comment:
auth = basic_authorization(user, passwrd) should be done before this function is called and auth passed to it instead of passing username and pw.